urllib3/util/connection.py

```diff
@@ -1,16 +1,19 @@
 from __future__ import absolute_import
 import socket
+from urllib3.util.rfc6555 import (
+    happy_eyeballs_algorithm
+)
 from .wait import wait_for_read
 from .selectors import HAS_SELECT, SelectorError
 
+ENABLE_HAPPY_EYEBALLS = True
```

Comment:
It's used in `create_connection`, I wanted to give users the ability to disable Happy Eyeballs if they wanted.  Should that be a parameter instead?